src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -318,10 +273,12 @@ function PaymentMethodList({
                     icon={item.icon}
                     disabled={item.disabled}
                     displayInDefaultIconColor
+                    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
                     iconHeight={item.iconHeight || item.iconSize}
```

Comment:
I think we can use `??` here - iconHeight should probably never be 0 - it just does not make sense(same for `iconWidth`)

Reply:
And if it is, we could still (kind oooof) argue that that if the caller requested 0, they want 0. We should bias towards the interpretation that `0` and `''` are values like all others. Hopefully one day this will be a fact in the whole codebase.

Reply:
If this sounds contradictive to what I said earlier, I'm sorry... I gave this some more thought and this is my current perspective:

- We should always **remember** that `??` and `||` have important differences in their semantics
- We should be careful to catch up fragments of the codebase that rely on the convention to treat `''` (or `[]` or `0` ...) as "not set" and act accordingly
- ...but without any clues towards such interpretation we should bias towards assuming that the mentioned values are not special at all

Reply:
@filip-solecki @blazejkustra Is this summary worth putting in the TS migration guidelines?  😜